src/components/data/data-utils.js

```diff
@@ -763,6 +779,138 @@ export const EntityEditor = ({ entityType, entityName, attributeName, attributeV
   ])
 }
 
+export const MultipleEntityEditor = ({ entityType, entityNames, attributeNames, entityTypes, workspaceId: { namespace, name }, onDismiss, onSuccess }) => {
```

Comment:
This is very similar to `SingleEntityEditor` (formerly `EntityEditor`). At first, I moved `Modal` out of `EntityEditor` and implemented this component as the attribute input and `EntityEditor`. However, the number of props needed for `EntityEditor` to support both cases started to feel unwieldy, so I copied much of `EntityEditor`'s code into this component.

Reply:
Yea it's not clear to me how well common code could be factored out in this case either.